skrub/_expressions/_parallel_coord.py

```diff
@@ -90,14 +100,21 @@ def _prepare_obj_column(col):
     }
 
 
-def _prepare_numeric_column(col, log_scale):
+def _prepare_numeric_column(col, log_scale, is_int):
     vals = col.to_numpy()
     if log_scale:
         vals = np.log(vals)
     min_val, max_val = np.nanmin(vals), np.nanmax(vals)
     tickvals = np.linspace(min_val, max_val, 10).tolist()
-    if pd.api.types.is_integer_dtype(col):
-        ticktext = [str(int(np.round(np.exp(v)))) for v in tickvals]
+    if pd.api.types.is_integer_dtype(col) or is_int:
+        if log_scale:
+            tickvals = np.exp(tickvals)
+        tickvals = np.unique(np.round(tickvals).astype("int"))
+        tickvals_label_space = tickvals.tolist()
+        if log_scale:
+            tickvals = np.log(tickvals)
```

Comment:
```suggestion
            # cannot make inplace log since `tickval` is of integral type
            tickvals = np.log(tickvals)
```

Reply:
Here tickvals is an array with only 10 elements, so I would prefer not adding the mental charge of having to think about the implications of in-place computation and the use of `out=input_buffer` because the optimization is not worth it for 10 elements. what do you think?